bin/edgex-docker-launch.sh

```diff
@@ -12,22 +12,22 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 
-# Usage: bin/edgex-docker-launch.sh [redis]
+# Usage: bin/edgex-docker-launch.sh [mongo]
 #
-# By default download the Mongo based Docker Compose file and attempt to start EdgeX. If the redis
-# option is used, download the Redis based Docker Compose file and attemp to start EdgeX.
+# By default download the Redis based Docker Compose file and attempt to start EdgeX. If the mongo
+# option is used, download the Mongo based Docker Compose file and attemp to start EdgeX.
 #
 # To override the compose file entirely set the COMPOSE_FILE_PATH environment variable to the full
 # pathname of the compose file you want to use.
 
-GITHUB_PATH=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/edinburgh/compose-files/
+GITHUB_PATH=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/fuji/compose-files/
 VERSION=1.0.1
 
 if [ "x$COMPOSE_FILE_PATH" = x ]; then
-    if [ $# -ne 0 ] && [ "$1" = "redis" ]; then
-        COMPOSE_FILE=docker-compose-redis-edinburgh-no-secty-$VERSION.yml
+    if [ $# -ne 0 ] && [ "$1" = "mongo" ]; then
+        COMPOSE_FILE=docker-compose-mongo-geneva-no-secty.yml
     else
-        COMPOSE_FILE=docker-compose-edinburgh-no-secty-$VERSION.yml
+        COMPOSE_FILE=docker-compose-redis-geneva-no-secty.yml
```

Comment:
Since we're on master, I suggest we switch over to the geneva tag. The new Geneva files are part of [developer-scripts PR #218](https://github.com/edgexfoundry/developer-scripts/pull/218).

Reply:
We don't cut Geneva tags/names until we do the actual release.

https://github.com/edgexfoundry/developer-scripts/pull/218#pullrequestreview-355425142